### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by PythonCoderAS**
- Added shields
- Fixed headings
- Sentences should start with capitalization except for filenames

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -4,16 +4,20 @@ asyncprawcore
 =============
 
 .. image:: https://img.shields.io/pypi/v/asyncprawcore.svg
-           :alt: Latest asyncprawcore Version
-           :target: https://pypi.python.org/pypi/asyncprawcore
-.. image:: https://travis-ci.org/praw-dev/asyncprawcore.svg?branch=master
-           :target: https://travis-ci.org/praw-dev/asyncprawcore
-.. image:: https://coveralls.io/repos/github/praw-dev/asyncprawcore/badge.svg?branch=master
-           :target: https://coveralls.io/github/praw-dev/asyncprawcore?branch=master
+   :alt: Latest Asyncpraw Version
```
Comment (LilSpazJoekp) [inline README.rst:7]: This should be `asyncprawcore`

Comment (PythonCoderAS) [reply README.rst:7]: I see, I thought I made the neccesary changes.

File: README.rst [hunk 2]
```diff
@@ -4,17 +4,21 @@ asyncprawcore
 =============
 
 .. image:: https://img.shields.io/pypi/v/asyncprawcore.svg
-           :alt: Latest asyncprawcore Version
```
Comment (vikramaditya91) [inline README.rst:7]: Why this change? It is according to PEP8 or something?

Comment (PythonCoderAS) [reply README.rst:7]: This was copied from `README.rst` in praw-dev/praw.

**Comment by PythonCoderAS** [comment]
@vikramaditya91 Changelog entries should only include changes in the code. Documentation updates do not warrant a new entry.

File: README.rst [hunk 3]
```diff
@@ -59,14 +63,14 @@ Save the above as ``trophies.py`` and then execute via:
...
 .. code-block:: console
 
-   python trophies.py
+   python3 trophies.py
 
 Additional examples can be found at:
 https://github.com/praw-dev/asyncprawcore/tree/master/examples
 
 
-Depending on asyncprawcore
```
[the remark this answers is not in the document]
Comment (PythonCoderAS) [reply README.rst:68]: I see, then I'll undo it.

File: README.rst [hunk 4]
```diff
@@ -59,14 +63,14 @@ Save the above as ``trophies.py`` and then execute via:
 
 .. code-block:: console
 
-   python trophies.py
+   python3 trophies.py
```
[the remark this answers is not in the document]
Comment (PythonCoderAS) [reply README.rst:66]: I see.
